Fixes #22813: alert word added on remove banner

kbatavia12 opened the pull request:
Alert word added on the banner when removing.
Eg: If the alert word "fintech" is removed, the banner reads 
> Alert Word 'fintech' removed successfully!

The screenshot shows the working.

Fixes: #22813 

![alert_remove](https://user-images.githubusercontent.com/42770054/192239123-e0837e3c-002c-4d5a-b886-4923b75ba25d.jpg)

**Self-review checklist**
- [x] Ran backend tests
- [x] Tested with puppeteer
- [x] Tested with node
- [x] Lint fix


Working
- [x] Visual appearance of the changes.
- [x] Responsiveness and internationalization.
- [x] Strings and tooltips.
- [x] End-to-end functionality of buttons, interactions and flows.
- [x] Corner cases, error conditions, and easily imagined bugs.

alya commented:
Please implement my comment and use ", not ', around the alert word.

I recommend going through https://zulip.readthedocs.io/en/latest/git/index.html, as it's difficult to review your work when each update is pushed to a new PR.

Thanks!

kbatavia12 commented:
@alya, Implemented your comment. Could you help me clean up the commit history, I cleaned up today's commits but the ones made last time aren't showing up while rebasing.
Thanks!

Alert word added on the banner when removing.
Eg: If the alert word "fintech" is removed, the banner reads 
> Alert Word 'fintech' removed successfully!

The screenshot shows the working.

Fixes: #22813 


![alert_remove](https://user-images.githubusercontent.com/42770054/192872562-32b748c3-6a0a-4d7b-9d9e-0ab07d6691ff.jpg)

**Self-review checklist**
- [x] Ran backend tests
- [x] Tested with puppeteer
- [x] Tested with node
- [x] Lint fix


Working
- [x] Visual appearance of the changes.
- [x] Responsiveness and internationalization.
- [x] Strings and tooltips.
- [x] End-to-end functionality of buttons, interactions and flows.
- [x] Corner cases, error conditions, and easily imagined bugs.

alya commented:
If you are having trouble with the commit history, you can [post a question](https://zulip.readthedocs.io/en/latest/contributing/asking-great-questions.html) in the #git help stream in the development community.

Please comment here once you've cleaned up your commits. Thanks!

kbatavia12 commented:
@alya, This PR just closed while cleaning up the commits, can you reopen it?

alya commented:
Not, it doesn't look like I can reopen it.
